web/datapipeline/projector.go

```diff
@@ -74,3 +78,11 @@ func (p *projector) Project(dataCollectedEvent *DataCollectedEvent) error {
 		return nil
 	})
 }
+
+func payloadDecoder(payload datatypes.JSON) *json.Decoder {
```

Comment:
what do you think about renaming this to `getPayloadDecoder`?

Reply:
good for me if it makes it clearer :wink: